plugin.json

```
@@ -3,7 +3,7 @@
     "name": "Cloud Apps",
     "description": "Cloud Apps Registry and API proxy.",
     "version": "0.1.0",
-    "min_server_version": "5.26.0",
+    "min_server_version": "5.35.0",
```

Per https://community-release.mattermost.com/core/pl/qz9gedh74jrufkmd7oaobh11ye the April 21th Cloud release wont be based on v5.35.0. Given that we target that release (right?) we need to stick to `5.34.0`

---

Well, we do depend on a later build, it appears, so don't we have a bigger problem then? Or do we continue to compile with 5.35, but _require_ 5.34 in JSON?

---

Yes, we can require 5.34 here
```suggestion
    "min_server_version": "5.34.0",
```